bedrock/settings/static_media.py

```diff
@@ -1277,6 +1277,8 @@
     },
     'firefox_firstrun_quantum': {
         'source_filenames': (
+            'js/base/mozilla-traffic-cop.js',
+            'js/firefox/firstrun/experiment-firstrun-email-first.js',
```

Comment:
These tow files should be in their own bundle, and then included in the `{% block experiments %}` block so that they excecute in the `<head>` of the page for a faster redirect.

Reply:
We should also wrap the experiment JS in a switch, so we can toggle the experiment on/off independently of a deployment. e.g.

```
{% block experiments %}
  {% if switch('experiment-firstrun-email', ['en-US']) %}
    {% javascript 'experiment-firstrun-email' %}
  {% endif %}
{% endblock %}
```

Note: the `en-US` locale bit means the bundle will only be included for that locale.

Reply:
Done